### DISCUSSION TITLE
ScriptReference/PlayModeStateChange

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#565] [Notes]
# ScriptReference/PlayModeStateChange

https://docs.unity3d.com/ScriptReference/PlayModeStateChange.html

**Comment by joonjoonjoon** [comment]
Subscribing to this event with scripting domain reload enabled (which is the default) seems to result in only getting the event with ExitingEditMode as parameter, but not getting EnteredPlayMode.  Observed in 2021 LTS
